Source/NSOutlineView.m

```diff
@@ -1815,14 +2035,32 @@ - (id) _objectValueForTableColumn: (NSTableColumn *)tb
 {
   id result = nil;
 
-  if ([_dataSource respondsToSelector:
-    @selector(outlineView:objectValueForTableColumn:byItem:)])
+  // If we have content binding the data source is used only
+  // like a delegate
+  NSDictionary *info = [GSKeyValueBinding infoForBinding: NSValueBinding forObject: tb];
```

Comment:
Looking at the binding of another object looks wrong to me. This code (and other similar one) belongs into NSTableColumn. I don't quite understand how it should be packaged up there, though.

Reply:
The issue here is this.   I believe that NSOutlineView on macOS works a bit differently than the one we have implemented.   Ours preloads the visible items and then gets them progressively as the items are expanded.   As such it is not always getting the self.whateverProperty pattern of stuff from the items.   Sometimes it is necessary to get things as shown in that method.   I believe it can be made more elegant, but I am not sure how at this point.

Reply:
Ahhh, yes, I will do that.   I see what you mean.

Reply:
I added this method.  It seems to clean things up a lot.